<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glaxnimate::gui::font::ExternalFontWidget</class>
 <widget class="QWidget" name="glaxnimate::gui::font::ExternalFontWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>421</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File/URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_url"/>
     </item>
     <item>
      <widget class="QPushButton" name="button_load">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset theme="download">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Open...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progress_bar"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_data" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="20,5,5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Font Style</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="view_fonts">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="view_style"/>
      </item>
      <item row="1" column="2">
       <widget class="glaxnimate::gui::font::FontSizeWidget" name="size_widget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glaxnimate::gui::font::FontSizeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/font/font_size_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>glaxnimate::gui::font::ExternalFontWidget</receiver>
   <slot>url_from_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>731</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_load</sender>
   <signal>clicked()</signal>
   <receiver>glaxnimate::gui::font::ExternalFontWidget</receiver>
   <slot>load_url()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_url</sender>
   <signal>textChanged(QString)</signal>
   <receiver>glaxnimate::gui::font::ExternalFontWidget</receiver>
   <slot>url_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>url_from_file()</slot>
  <slot>load_url()</slot>
  <slot>url_changed(QString)</slot>
 </slots>
</ui>
